spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/connection/SocketSupportTests.java

```diff
@@ -518,7 +518,7 @@ public void testNioClientAndServerSSL() throws Exception {
 	}
 
 	@Test
-	@EnabledOnJre(JRE.JAVA_8)
+	@EnabledOnJre(JRE.JAVA_8) // Need to wait for write complete on Java 14
```

Comment:
@garyrussell , what does it mean?
Is there a way to do something similar what you have with that `if (System.getProperty("java.version").startsWith("1.8")) {` ?

Reply:
I fixed it in the next commit.